.ebextensions/enable_mod_deflate.conf

```diff
@@ -0,0 +1,26 @@
+# mod_deflate configuration
+<IfModule mod_deflate.c>
+  # Restrict compression to these MIME types
+  AddOutputFilterByType DEFLATE text/plain
+  AddOutputFilterByType DEFLATE text/html
+  AddOutputFilterByType DEFLATE application/xhtml+xml
+  AddOutputFilterByType DEFLATE text/xml
+  AddOutputFilterByType DEFLATE application/xml
+  AddOutputFilterByType DEFLATE application/xml+rss
+  AddOutputFilterByType DEFLATE application/x-javascript
+  AddOutputFilterByType DEFLATE text/javascript
+  AddOutputFilterByType DEFLATE text/css
+  # Level of compression (Highest 9 - Lowest 1)
+  DeflateCompressionLevel 9
+  # Netscape 4.x has some problems.
```

Comment:
For the record, I don't really care about Netscape support ;)

Reply:
OA + DISJ goals we gotta take everyone into consideration

Reply:
😁